sql/2023-11-02-individual-scoring-view.sql

```diff
@@ -0,0 +1,8 @@
+CREATE VIEW individual_scoring AS
+SELECT entity_id,
+       CASE WHEN score_type = 'current' THEN score END AS current,
+       CASE WHEN score_type = 'recency' THEN score END AS recency,
+       CASE WHEN score_type = 'traffic' THEN score END AS traffic,
+       CASE WHEN score_type = 'typicality' THEN score END AS typicality,
+       CASE WHEN score_type = 'similarity' THEN score END AS similarity
+FROM score;
```

Comment:
this doesn't work does it? there's N scores for each entity. We want a table that has:

entity_id, current_score, recency_score, traffic_score, typicality_score, similarity_score, avg_score

...columns

Reply:
I'm not sure, however last time I checked, there is only one value to each score type. I will test it further but I believe it does the right thing. 

Or is it that you want it to be automated in case there is new score types ? In that case I will work on that.

In any case, this is a draft and there is still a few things missing, I'll do my best to correct everything.

Reply:
please provide example outputs and steps you took to test it.

also, please rework the title; you'll be creating a lot of sql views so "Creation of the sql view" is not going to be helpful when we look back at the history.

Reply:
reworked the title, is it better ?

Also regarding example outputs, do you want it in comments or in the code ?

Reply:
give me a couple sample lines here posted in the description of the PR. as well, keep a checklist of tasks done within this PR in your PR description.

Reply:
first line of the csv file : entity_id,current,recency,traffic,typicality,similarity,avg
This means the table has a correct structure.

Had a case like this : 
1bc6def7-xxxx,1.0,,,,,,1.0
1bc6def7-xxxx,,0.8294036718602102,,,,0.8294036718602102
1bc6def7-xxxx,,,0.08711416212970638,,,0.08711416212970638
1bc6def7-xxxx,,,,0.0012760527435133986,,0.0012760527435133986

Corrected the SQL file and got the correct output which is supposed to look like this :
1bc6def7-xxxx,1.0,0.8294036718602102,0.08711416212970638,0.0012760527435133986,0.0,1.9177938867334299

Reply:
_here posted in the description of the PR._

Reply:
done